### PR TITLE
Merge initial build

### LINKED ISSUE #1: Initial Build
**Description**
Access Clearbit Enrichment API 

**Requirements**
- [x] Build methods to access `combined`, `email`, or `company` endpoints.
- [x] Keep under [rate limit of 600 requests/min](https://clearbit.com/docs#rate-limiting)
- [x] Authenticate via API key
- [x] Provide `output` parameter enabling either:
  - [x] `Separate` - The data provided by Clearbit is the sole output
  - [x]  `Append` - The data provided by Clearbit is appended to the original records

**Relevant Links**
https://clearbit.com/docs#enrichment-api

**Possible config**
```
"config":{
  "api":"email"
  "apiKey":"158025712481249"
  "field":"emailAddress",
  "output":"append"
}
```

### PR DESCRIPTION
**Pull request by noahbass**
Adds support for accessing Clearbit Enrichment API endpoints (`combined`, `email`, and `company`)

Closes #1 ⚡️

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,10 +1,85 @@
 import { Activity, singleS3StreamInput, singleS3StreamOutput } from 'aries-data';
+import clearbit from 'clearbit';
 
 export default class ClearbitEnrichment extends Activity {
-
     @singleS3StreamInput()
     @singleS3StreamOutput()
-    async onTask(activityTask, config, lastExecuted) {
-        throw new Error('onTask not implemented');
+    async onTask(activityTask, config) {
```
Comment (trong-le) [inline lib/index.js]: add executionDate

File: README.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+     'google_apps',
+     'hubspot',
+     'nginx',
+     'hotjar',
+     'mixpanel' ] }
+```
+
+## Tests
+
+Run tests with `npm test`. Check coverage with `npm run coverage`.
```
Comment (trong-le) [inline README.md]: add license

File: README.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+## Methods
+
+`combined(email, extraParams, output, apiKey)` - Looks up a person and company with an email address and other optional parameters
+
+Person data is returned to the `person` object, company data is returned to the `company` object.
+
+`email(email, extraParams, output, apiKey)` - Looks up a person with an email address and other optional parameters
+
+`company(domain, extraParams, output, apiKey)` - Looks up a company with a domain and other optional parameters
+
```
Comment (trong-le) [inline README.md]: add Configuration and explain available fields

**Comment by ryw** [comment]
no more work on aries

**Pull request closed without merging**